packages/optimizer/spec/transformers/valid/RewriteAmpUrls/adds_esm_and_preloads/input.html

```diff
@@ -0,0 +1,18 @@
+<!--
+{
+  "esmModulesEnabled": true,
+  "preloadEnabled": true
```

Comment:
This isn't getting recognized. Why?

Reply:
This is passed into the transformer's `transform()` method as the second `params` argument and thus needs to be read from there. You can see that `params.esmModulesEnabled` is already being processed in that method.